shell/platform/windows/compositor_opengl.cc

```diff
@@ -154,7 +154,12 @@ bool CompositorOpenGL::Present(const FlutterLayer** layers,
                        GL_NEAREST            // filter
   );
 
-  return engine_->view()->SwapBuffers();
+  if (!engine_->surface_manager()->SwapBuffers()) {
+    return false;
+  }
+
+  engine_->view()->OnFramePresented();
+  return true;
```

Comment:
Should this be refactored into a separate function? It looks like this and the below addition to this file are identical.

Reply:
They are identical, but I think the logic is short enough that it's okay to repeat. I don't feel super strongly about this though, let me know if you do and I'd be happy to update this.

Reply:
Just a thought. I don't think it's too important either way.